crates/fhe/examples/trbfv_add.rs

```diff
@@ -39,40 +44,38 @@ fn print_notice_and_exit(error: Option<String>) {
 }
 
 fn main() -> Result<(), Box<dyn Error>> {
-    // Parameters
+    // Parameters for threshold BFV with l-BFV integration
     let degree = 2048;
     let plaintext_modulus: u64 = 4096;
     let moduli = vec![0xffffee001, 0xffffc4001, 0x1ffffe0001];
 
-    // This executable is a command line tool which enables to specify
-    // trBFV summations with party and threshold sizes.
+    // Command line argument parsing
     let args: Vec<String> = env::args().skip(1).collect();
 
-    // Print the help if requested.
+    // Print help if requested
     if args.contains(&"-h".to_string()) || args.contains(&"--help".to_string()) {
         print_notice_and_exit(None)
     }
 
     let mut num_summed = 1;
-    let mut num_parties = 10;
-    let mut threshold = 7;
+    let mut num_parties = 5;
```

Comment:
Let's make the default 2/3 for the sake of speed.

Reply:
will change, 1s